yarn.lock

```diff
@@ -318,6 +318,8 @@ __metadata:
     eslint-plugin-eslint-comments: ^3.2.0
     eslint-plugin-import: 2.25.4
     eslint-plugin-jsdoc: ^37.9.4
+    fp-ts: 2.11.4
+    io-ts: 2.2.16
     mocha: ^9.2.1
```

Comment:
Since I removed `IsoRegion`, these two packages are no longer needed so I did a `yarn remove` for both packages. They were removed in the package.json but not here. Is this sus? @anotherminh @michaelfarrell76